conf/validation-config/OTHER/ers-other-notional-validation.conf

```diff
@@ -106,8 +106,8 @@ ers-other-notional-validation-config{
 
     {
       column = "G"
-      cellName = "7.National Insurance number (if applicable)"
-      mandatory = false
+      cellName = "7.National Insurance number"
```

Comment:
please add space for consistency after 7.

Reply:
Similar to ers-checking-frontend seems like the whole file is set up this way, though am happy to change it

Reply:
please ignore as well, thanks 👍

Reply:
https://github.com/hmrc/ers-file-validator/pull/48/commits/d1ae8a63c65f3cf0dd49f40f09be471cf7dd28a7

Added changes